### ISSUE TITLE
CSS validation errors

### ISSUE THREAD

**Issue by Lenzo** [Lenzo/dsn1675-assignment-1#1]
Do  I need to worry about this? It seems to just not like my colours, is it just because they are newer colours? Is it wrong to use the names for colours?

![screen shot 2015-09-17 at 6 13 51 pm](https://cloud.githubusercontent.com/assets/10464664/9947417/f1294576-5d67-11e5-9cc4-1171f56225fc.png)

Thanks
Len

**Comment by Lenzo** [comment]
Hey @thomasjbradley I'm still getting these errors.. Is it ok to use colour names?

**Comment by thomasjbradley** [comment]
Yep those are okay, but `rebeccapurple` is a really, really new colour and is likely to not work when we do browser testing.

There are two errors related to 'Playfair' that should be fixed.
